core/src/main/java/org/apache/iceberg/avro/SchemaToType.java

```diff
@@ -169,18 +184,15 @@ public Type primitive(Schema primitive) {
         return Types.DecimalType.of(
             ((LogicalTypes.Decimal) logical).getPrecision(),
             ((LogicalTypes.Decimal) logical).getScale());
-
       } else if (logical instanceof LogicalTypes.Date) {
         return Types.DateType.get();
-
       } else if (
           logical instanceof LogicalTypes.TimeMillis ||
-          logical instanceof LogicalTypes.TimeMicros) {
+              logical instanceof LogicalTypes.TimeMicros) {
```

Comment:
Minor: this file also has lots of formatting changes that should be reverted before committing.

Reply:
ack, I did not intentionally do these changes , formatting style from repo did these changes. I suggest same in our internal repo :) , I will revert these changes.